package.json

```diff
@@ -91,7 +91,7 @@
     "lint-staged": "^7.2.0",
     "module-alias": "^2.1.0",
     "openzeppelin-solidity": "^2.0.0",
-    "set-protocol-utils": "^1.0.0-beta.22",
+    "set-protocol-utils": "^1.0.0-beta.23",
```

Comment:
Let's use 0.24 since its the latest and to ensure no collisions between my PR and yours

Reply:
+1